### PR TITLE
change timeout in massa-client test scenario (cautious bassoon)

### PR DESCRIPTION
**Pull request by AureliaDolo**

### CODE REVIEW

**Review by adrien-zinger — APPROVED** [review]

**Comment by AureliaDolo** [comment]
bors merge

File: massa-client/src/tests/scenarios.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@ use serde::de::DeserializeOwned;
 use serial_test::serial;
 use std::{thread::JoinHandle, time::Duration};
 
-const TIMEOUT: u64 = 10;
+const TIMEOUT: u64 = 30;
 const TENTATIVES: u64 = 5;
```
Comment (damip) [inline massa-client/src/tests/scenarios.rs:9]: ATTEMPTS

**Comment by yvan-sraka** [comment]
Can we rename the PR to match what she does? (change timeout in `massa-client` test scenario)

**Comment by AureliaDolo** [comment]
Main has been merged here #1979 so this PR is useless

**Pull request closed without merging**